scripts/game_structure/load_cat.py

```diff
@@ -95,32 +70,64 @@ def json_load():
                         eye_colour=cat["eye_colour"] if cat["eye_colour"] not in ["BLUEYELLOW", "BLUEGREEN"] else "BLUE",
                         pelt=new_pelt,
                         loading_cat=True)
+            if cat["eye_colour2"] == "BLUE2":
+                new_cat.eye_colour2 = "COBALT"
             new_cat.eye_colour2 = cat["eye_colour2"] if "eye_colour2" in cat else None
+            if cat["eye_colour"] == "BLUEYELLOW":
+                new_cat.eye_colour2 = "YELLOW"
+            elif cat["eye_colour"] == "BLUEGREEN":
+                new_cat.eye_colour2 = "GREEN"
             new_cat.age = cat["age"]
             new_cat.genderalign = cat["gender_align"]
             new_cat.backstory = cat["backstory"] if "backstory" in cat else None
-            new_cat.birth_cooldown = cat[
-                "birth_cooldown"] if "birth_cooldown" in cat else 0
+            new_cat.birth_cooldown = cat["birth_cooldown"] if "birth_cooldown" in cat else 0
             new_cat.moons = cat["moons"]
-            new_cat.trait = cat["trait"]
+            if cat["trait"] in ["clever", "patient", "empathetic", "altruistic"]:
+                new_cat.trait = "compassionate"
+            else:
+                new_cat.trait = cat["trait"]
             new_cat.mentor = cat["mentor"]
             new_cat.former_mentor = cat["former_mentor"] if "former_mentor" in cat else []
             new_cat.patrol_with_mentor = cat["patrol_with_mentor"] if "patrol_with_mentor" in cat else 0
             new_cat.mentor_influence = cat["mentor_influence"] if "mentor_influence" in cat else []
             new_cat.paralyzed = cat["paralyzed"]
+            if new_cat.paralyzed:
+                new_cat.get_permanent_condition("paralyzed")
             new_cat.no_kits = cat["no_kits"]
             new_cat.exiled = cat["exiled"]
-            new_cat.age_sprites['kitten'] = cat["spirit_kitten"]
-            new_cat.age_sprites['adolescent'] = cat["spirit_adolescent"]
-            new_cat.age_sprites['young adult'] = cat["spirit_young_adult"]
-            new_cat.age_sprites['adult'] = cat["spirit_adult"]
-            new_cat.age_sprites['senior adult'] = cat["spirit_senior_adult"]
-            new_cat.age_sprites['elder'] = cat["spirit_elder"]
+            new_cat.cat_sprites['kitten'] = cat["sprite_kitten"] if "sprite_kitten" in cat else cat["spirit_kitten"]
```

Comment:
The else statement should be a default variable + this if else doesn't do anything. If "sprite_kitten" is not in the cat dictionary, this would lead to an error. 

This also applies to the other spirit setting variables